docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md

```diff
@@ -15,39 +15,30 @@ Hardware requirements:
 <a name="on_Windows"></a>
 ## Installing Intel&reg; HAXM on Windows&reg;
 
-Supported Windows&reg; versions: 8.1/8/7 (32/64-bit), Vista (32/64-bit), XP SP2 or higher (32-bit only)
+Supported Windows&reg; versions: 10/8.1/8/7 (32/64-bit), Vista (32/64-bit), XP SP2 or higher (32-bit only)
 
-**HAXM is installed automatically as part of the Tizen Studio**. If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/6.0.4/win/](http://download.tizen.org/sdk/haxm/6.0.4/win/) and download the Windows installer package.
+**HAXM is installed automatically as part of the Tizen Studio**. If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/7.0.0/win/](http://download.tizen.org/sdk/haxm/7.0.0/win/) and download the Windows installer package.
 
 **Table: Windows installer package**
 
-| Windows installer                        | File description | File size | MD5 checksum                     |
+| Windows installer                        | File description | File size | SHA256SUM                     |
 |----------------------------------------|----------------|---------|--------------------------------|
-| [IntelHAXM_6.0.4.exe](http://download.tizen.org/sdk/haxm/6.0.4/win/IntelHAXM_6.0.4.exe) | System driver    | 2.66 MB   | bc05f881d9282ed0914b3e0bf3c96e89 |
+| [IntelHAXM_7.0.0.exe](http://download.tizen.org/sdk/haxm/7.0.0/win/IntelHAXM_7.0.0.exe) | System driver    | 2.89 MB   | 1587f7ae42e11c7b22dc674fd776ce252b6a19152e7ca416a748e2bdbf2b36c5 |
 
 To install Intel&reg; HAXM:
 
 1. Run the downloaded installer and accept the UAC prompt.
 
    ![UAC prompt](./media/hardware_uac_prompt.png)
 
-2. Click **Next** to start the installation.  
+2. Click **Install** to start the installation.  
 ![Start installation](./media/hardware_start_instal.png)  
 You can access the documentation at any time by clicking **Intel&reg; HAXM Documentation**.
 
-3. Adjust the amount of RAM to be allocated to Intel&reg; HAXM.
-
-   ![Adjust RAM](./media/hardware_ram.png)
-
-   The installer also functions as a configuration tool for Intel&reg; HAXM. To change the memory settings later, run the installer again.
-
-   > **Note**  
-   > If you change the Intel&reg; HAXM memory settings for the emulator, you must restart the Intel&reg; HAXM and the emulator. Emulators currently running continue to use the previous memory setting.
-
-4. Confirm your Intel&reg; HAXM memory allocation settings and click **Install**.  
+3. **Install** will trigger the installation  
 ![Confirm installation](./media/hardware_instal.png)
 
-5. When the Intel&reg; HAXM installation is finished, click **Finish** to exit the installer.  
+4. When the Intel&reg; HAXM installation is finished, click **Finish** to exit the installer.  
```

Comment:
Rephrase as following: 
Click **Finish** to exit the installer, once the Intel&reg; HAXM installation is finished.

Reason being - We always follow the rule of Action first, reason/outcome later.